manim/mobject/coordinate_systems.py

```diff
@@ -403,16 +470,18 @@ class ThreeDAxes(Axes):
 
     def __init__(
         self,
-        x_range=(-6, 6, 1),
-        y_range=(-5, 5, 1),
-        z_range=(-4, 4, 1),
-        x_length=config.frame_height + 2.5,
-        y_length=config.frame_height + 2.5,
-        z_length=config.frame_height - 1.5,
-        z_axis_config=None,
-        z_normal=DOWN,
-        num_axis_pieces=20,
-        light_source=9 * DOWN + 7 * LEFT + 10 * OUT,
+        x_range: Optional[Union[Sequence[int], np.ndarray]] = (-6, 6, 1),
+        y_range: Optional[Union[Sequence[int], np.ndarray]] = (-5, 5, 1),
+        z_range: Optional[Union[Sequence[int], np.ndarray]] = (-4, 4, 1),
+        x_length: Optional[float] = config.frame_height + 2.5,
+        y_length: Optional[float] = config.frame_height + 2.5,
+        z_length: Optional[float] = config.frame_height - 1.5,
+        z_axis_config: Optional[dict] = None,
+        z_normal: Union[Sequence[float], np.ndarray] = DOWN,
```

Comment:
Isn't `Sequence[float]` sufficient here?
```suggestion
        z_normal: Sequence[float] = DOWN,
```

Reply:
No sometime we use `np.ndarray` for this

Reply:
Check the [typehint guidelines for manim](https://docs.manim.community/en/stable/contributing/documentation.html#adding-type-hints-to-functions-and-parameters) from the documentation. It says - coordinates have the typehint `Sequence[float]`. 

It is recommended to use `np.ndarray`s where ever you need to pass coordinates in manim, but in this case it can be a list or a tuple as well.